### PR TITLE
A solution in the issue #3325

### PR DESCRIPTION
**Pull request by Stergios13**
To begin with, i created a single function called archiveFile(filename, filePath) that takes 2 arguments. The first one declares the name of the file that has to be saved in jerkins and the second one declares the whole path of the file. In the post function (especially in line 690) i created a hash map that each key declares the filename and each value of the key declares the whole path of it. The function is called in a for-each loop after the hash map declaration.

PS1. i am not 100% sure if that works, but if it doesn't i'll be happy to contribute again
PS2: thanks for assigning me for this issue, i really wanted to contribute as it is my first open source contribution!

### CODE REVIEW

**Comment by llxia** [comment]
Thanks @Stergios13 for working on this issue. Please see https://github.com/adoptium/aqa-tests/blob/master/Contributing.md for commit guidelines.

**Comment by Stergios13** [comment]
I changed the file again listening to your tips. 
Additionally, i do not format the whole file, i just change the lines that need to be changed and yet it seems like i format the whole file.

**Comment by sophia-guo** [comment]
Running good https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/3586/

File: buildenv/jenkins/JenkinsfileBase [hunk 1]
```diff
@@ -685,18 +685,26 @@ def post(output_name) {
...
+
+            //Defining the files to be archived
+            def testenvFile = "aqa-tests/testenv/testenv.properties"
+            def shaFile = "aqa-tests/TKG/SHA.txt"
+            def AQACertFile = "aqa-tests/TKG/AQACert.log"
+
+            //call the archive function for each file
+            archiveFile(testenvFile)
+	    archiveFile(shaFile)
+            archiveFile(AQACertFile)
```
Comment (sophia-guo) [inline buildenv/jenkins/JenkinsfileBase:698]: As those filename is only used once may not need to define the variable. 

```
            archiveFile("aqa-tests/testenv/testenv.properties")
	    archiveFile("aqa-tests/TKG/SHA.txt")
            archiveFile("aqa-tests/TKG/AQACert.log")
```

Comment (smlambert) [reply buildenv/jenkins/JenkinsfileBase:698]: @Stergios13 - can you update your PR based on @sophia-guo's suggestion (pass the filenames directly, as we do not need to separately define variables for our purpose.

Comment (Stergios13) [reply buildenv/jenkins/JenkinsfileBase:698]: Sorry for being inactive i was busy due to university purposes. Of course i will during the day!

**Comment by smlambert** [comment]
FYI @llxia @sophia-guo - if @Stergios13 can update this quickly we can wait, otherwise suggest merging and having a followup PR to clean up.

#3394 is somewhat related.

**Review by sophia-guo — APPROVED** [review]
Minor issue could be updated later.

**Review by smlambert — APPROVED** [review]

**Pull request merged**